addons/stock_intrastat/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Stock Intrastat',
+    'category': 'Stock',
+    'description': """
+A module that add the stock management in intrastat reports.
+============================================================
+
+This module gives the details of the goods traded between the countries of
+European Union.""",
+    'depends': ['stock_account', 'account_intrastat', 'stock_dropshipping'],
+    'data': [
+        'views/stock_warehouse_view.xml',
+    ],
+    'auto_install': True,
```

Comment:
can't be auto installed with dropshipping module in the dependencies

Reply:
I can remove the dependency to the dropshipping module (only needed for `self.env.ref('stock_dropshipping.picking_type_dropship')`) or remove the `auto_install`.